src/sidebar/util/test/sentry-test.js

```diff
@@ -1,8 +1,6 @@
 import * as sentry from '../sentry';
 
 describe('sidebar/util/sentry', () => {
-  let fakeDocumentReferrer;
```

Comment:
This is a fake for `document.referrer` which used to be used by `sentry.js` but no longer is.